todo_app/viewmodel.py

```diff
@@ -22,11 +22,11 @@ def show_all_done_items(self):
     def recent_done_items(self):
         all_done_items = [item for item in self._items if item.status == 'Done']
         today = datetime.date.today()
-        recent_done_items = [item for item in all_done_items if item.lastmodifieddate >= today]    
+        recent_done_items = [item for item in all_done_items if item.lastmodifieddate.day >= today.day]    
```

Comment:
Similarly here, I think you should stick with your previous solution - comparing by day only breaks your tests (on _some_ days of the year) and breaks the functionality